### PR TITLE
Jayden

### PR DESCRIPTION
**Pull request by Jaydenmd**

### CODE REVIEW

**Comment by rastguelenianrn** [comment]
Your code looks like it works, so good job on that! Only thing Im wondering is if you can possibly remake the branch so you can have less commits? (right now there's 21, and it looks like you were reworking it alot, so maybe you can remake the branch and just have your final commit?)

File: Test/myTest2 [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from testing_base import *
+
+def aceTest():
+    one = ('Spade', 8)
+    two = ('ace')
+    
+    num_aces=0
+    non_ace_total=0
+    if num_aces > 0:
```
Comment (MidnightBlackHorse17) [inline Test/myTest2:9]: Great work Jayden! But like Chris mentioned, this might become problematic if Ace is a variable throughout the game.

Comment (Jaydenmd) [reply Test/myTest2:9]: Thanks! Yeah for sure, I would want to see how I could work my code with everyone else's for sure.

**Comment by Jaydenmd** [comment]
> Your code looks like it works, so good job on that! Only thing Im wondering is if you can possibly remake the branch so you can have less commits? (right now there's 21, and it looks like you were reworking it alot, so maybe you can remake the branch and just have your final commit?)

Response: "Thanks man! Yeah for sure, I'll see if I can do something about it with the many other commits haha

**Pull request merged**